sites/partners/src/pages/reset-password.tsx

```diff
@@ -35,9 +35,9 @@ const ResetPassword = () => {
 
     try {
       const user = await resetPassword(token.toString(), password, passwordConfirmation)
-      setSiteAlertMessage(t(`authentication.signIn.success`, { name: user.firstName }), "success")
       await router.push("/")
       window.scrollTo(0, 0)
+      setToast(t(`authentication.signIn.success`, { name: user.firstName }), "success")
```

Comment:
This might be one of the cases we want to remove. When following through the forgot password flow, I saw a flashing toast after resetting my password

Reply:
I agree that this should be removed. I'll make a note that this should be an alert instead